src/components/TitleBody/TitleBody.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const TitleBody = ({ title, body }) => {
+  
+    return (
+        <>
+            <h2 className="title">{title}</h2>
+            <div className="body" dangerouslySetInnerHTML = {{ __html: body }} />
+        </>
+        )
+    }
+
+  export default TitleBody;
```

Comment:
Looks like your code editor made this file weirdly spaced, we also need to be overly verbose since title and body could be over written

```suggestion
import React from 'react';

const TitleBody = ({ title, body }) => { 
  return (
      <>
          <h2 className="title-body__title">{title}</h2>
          <div className="title-body__body" dangerouslySetInnerHTML = {{ __html: body }} />
      </>
    )
}

export default TitleBody;
```